internal/linker/link_node_modules.bzl

```diff
@@ -56,12 +56,13 @@ def _link_mapping(label, mappings, k, v):
     else:
         return True
 
-def write_node_modules_manifest(ctx, extra_data = []):
+def write_node_modules_manifest(ctx, extra_data = [], mnemonic = ""):
```

Comment:
nit: maybe it should default to `None`, then later we could avoid having the extra `_%s` appear if mnemonic is unset? my understanding is that it's called from run_node with kwargs.get("mnemonic") which means None is the value that will be commonly passed here anyway

Reply:
Done